gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5-all.zip
```

Comment:
I'd revert to `5.6.2` possibly.
I'm also in favor for keeping Gradle 6.1 and just letting our users know that they need to update Gradle.

Reply:
I'm going to preserve 5.5 as it was the version used in 1.2.0  (https://github.com/Yelp/swagger-gradle-codegen/blob/1.2.0/gradle/wrapper/gradle-wrapper.properties#L3)